connaisseur/logging_wrapper.py

```diff
@@ -11,7 +11,10 @@ def __init__(self, app, log_level):
         self.logger = logging.getLogger(
             "connaisseurRequestLogger"
         )  # has no handler and hence propagates logs to root logger
-        self.logger.setLevel(log_level)
+        try:
+            self.logger.setLevel(log_level)
+        except ValueError:
```

Comment:
Would you want to log this misconfig?

Reply:
hm ... I see now that this is dead code ... a wrong log level already raises an exception in connaisseurs `__main__.py` ... so i guess i revert this back